homeassistant/components/html5/notify.py

```diff
@@ -540,6 +544,11 @@ def _push_message(self, payload, **kwargs):
                     _LOGGER.error("Error saving registration")
                 else:
                     _LOGGER.info("Configuration saved")
+            elif response.status_code != 201:
```

Comment:
Is this the same for every push notification implementation ? 

The PR description makes it sound like this was only tested on Chrome. 

I would expect it to check `response.status_code > 399`

Reply:
I did test it on Chrome as well, but it does appear that other codes can appear as mentioned [here](https://blog.pushpad.xyz/2022/04/web-push-errors-explained-with-http-status-codes/).